kythe/go/extractors/config/runextractor/gradlecmd/gradlecmd.go

```diff
@@ -52,6 +53,11 @@ func (g *gradleCommand) SetFlags(fs *flag.FlagSet) {
 }
 
 func (g gradleCommand) verifyFlags() error {
+	for _, key := range []string{"KYTHE_CORPUS", "KYTHE_ROOT_DIRECTORY", "KYTHE_OUTPUT_DIRECTORY", "JAVAC_EXTRACTOR_JAR", "REAL_JAVAC"} {
```

Comment:
Consider maybe defining constants for these someplace. It doesn't have to be in this PR, though.

Reply:
done